### ISSUE TITLE
'SNF' is not an exported object from 'namespace:liger'

### ISSUE THREAD

**Issue by kaizen89** [satijalab/seurat-wrappers#33]
I'm having this error (title of the issue) when trying to run `RunQuantileAlignSNF` and I can't find the cause of the issue
```
> sessionInfo()
R version 3.6.3 (2020-02-29)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Ubuntu 18.04.3 LTS

Matrix products: default
BLAS:   /usr/lib/x86_64-linux-gnu/atlas/libblas.so.3.10.3
LAPACK: /usr/lib/x86_64-linux-gnu/atlas/liblapack.so.3.10.3

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C               LC_TIME=fr_FR.UTF-8        LC_COLLATE=en_US.UTF-8    
 [5] LC_MONETARY=fr_FR.UTF-8    LC_MESSAGES=en_US.UTF-8    LC_PAPER=fr_FR.UTF-8       LC_NAME=C                 
 [9] LC_ADDRESS=C               LC_TELEPHONE=C             LC_MEASUREMENT=fr_FR.UTF-8 LC_IDENTIFICATION=C       

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] liger_0.5.0.9000     patchwork_1.0.0      Matrix_1.2-18        cowplot_1.0.0        SeuratWrappers_0.1.0
[6] dplyr_0.8.4          Seurat_3.1.4  
```
Any help?
Thanks

**Comment by KyriakisDimitrios** [comment]
The same error as @kaizen89 
Thanks

**Comment by dannyn66** [comment]
Same error using the pbmcsca data set from SeuratData.

I just ran the following:
data("pbmcsca")
pbmcsca <- NormalizeData(pbmcsca)
pbmcsca <- FindVariableFeatures(pbmcsca)
pbmcsca <- ScaleData(pbmcsca, split.by = "Method", do.center = FALSE)
pbmcsca <- RunOptimizeALS(pbmcsca, k = 20, lambda = 5, split.by = "Method")
pbmcsca <- RunQuantileAlignSNF(pbmcsca, split.by = "Method")

R version 3.6.1 (2019-07-05)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Antergos Linux

other attached packages:
 [1] liger_0.5.0.9000          patchwork_1.0.0           Matrix_1.2-17             cowplot_1.0.0             SeuratWrappers_0.1.0     
 [6] pbmcsca.SeuratData_3.0.0  pbmc3k.SeuratData_3.0.0   hcabm40k.SeuratData_3.0.0 SeuratData_0.2.1          Seurat_3.1.4

**Comment by lixin4306ren** [comment]
got same error.

**Comment by smmiba** [comment]
same error

**Comment by KyriakisDimitrios** [comment]
I just did not used the Wrapper in the end. Probably Liger updated the function and wrapper didn't udate it.
I used this and it worked. Then you can just transfer the projection to seurat object again
```
data.liger <- createLiger(sapply(ob.list, function(data) data[['RNA']]@counts[, colnames(data)]), remove.missing = F) 
data.liger <- liger::normalize(data.liger)
data.liger@var.genes <- var.genes
data.liger <- scaleNotCenter(data.liger)
data.liger <- optimizeALS(data.liger, k = k, lamda = lamda) 
data.liger <- quantileAlignSNF(data.liger, resolution = 1.2) 
data.liger <- runUMAP(data.liger, n_neighbors = n_neighbors, min_dist = 0.3) 

seurat_obj = ligerToSeurat(data.liger, use.liger.genes = T)
```

**Comment by mcap91** [comment]
Hi, I answered this here

https://github.com/MacoskoLab/liger/issues/155

They have updated liger. you need to use liger 0.4.2 to have wrapper functionality, until the wrappers are updated.

**Comment by mojaveazure** [comment]
This was addressed in #41
